.github/instructions/website.blog.instructions.md

```diff
@@ -0,0 +1,343 @@
+---
+applyTo: website/blog/**/index.md
+---
+
+# AKS Blog Post Content Guidelines
+
+> **Scope**: File-specific patterns for blog post Markdown files.  
+> **Module Context**: See `website/AGENTS.md` for architecture and workflows.  
+> **Repo Standards**: See `.github/copilot-instructions.md` for general conventions.
+
+## Front Matter (Required)
+
+Every blog post MUST include:
+
+```yaml
+---
+title: "Descriptive Post Title"
+date: YYYY-MM-DD
+description: "SEO-optimized summary (150-160 characters)"
+authors: [author-key]  # From blog/authors.yml
+tags: [tag1, tag2]     # From blog/tags.yml
+---
+```
+
+**Validation Checklist**:
+
+- [ ] `title`: Clear, descriptive, under 60 chars
+- [ ] `date`: Valid format, can be future-dated for future publishing
+- [ ] `description`: 150-160 chars, includes keywords
+- [ ] `authors`: Valid keys from `blog/authors.yml`
+- [ ] `tags`: Valid keys from `blog/tags.yml`
+
+## Content Structure
+
+### Required Pattern
+
+```markdown
+---
+title: "..."
+date: YYYY-MM-DD
+description: "..."
+authors: [...]
+tags: [...]
+---
+
+[Opening paragraph: Hook the reader, set context, preview content]
+
+<!-- truncate -->
+
+![Hero Image](./hero-image.png)
+
+## Section 1: Problem/Context
+[Describe the challenge or background...]
+
+## Section 2: Solution/Details
+[Explain the approach or feature...]
+
+## Section 3: Implementation
+[Step-by-step guide or examples...]
+
+## Conclusion
+[Summarize key takeaways...]
+```
+
+### Critical Elements
+
+1. **Truncation marker**: `<!-- truncate -->` after 2-3 intro paragraphs (shows on listing page)
+2. **Hero image**: Use `./image.png` for same-directory assets
+3. **Heading hierarchy**: H2 (`##`) for major sections, H3 (`###`) for subsections
+4. **Alt text**: All images MUST have descriptive alt text
+
+## Writing Style
```

Comment:
Can we incorporate instructions from the Microsoft style guide: https://learn.microsoft.com/en-us/style-guide/welcome/